### DISCUSSION TITLE
从 Kubernetes 中的对象谈起 · /kubernetes-object-intro

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#535] [Announcements]
https://draveness.me/kubernetes-object-intro

**Comment by gtt116** [comment]
图片的配色很赞

**Comment by fubai** [comment]
1.“Spec - 对象接口”：第一句话，多了一个“就是”
2.“Spec - Pod”：“这些容器会共享网络和进程”，应该改成“这些容器会共享网络和存储”

**Comment by draveness** [comment]
> 1.“Spec - 对象接口”：第一句话，多了一个“就是”
> 2.“Spec - Pod”：“这些容器会共享网络和进程”，应该改成“这些容器会共享网络和存储”

修复了，多谢

**Comment by kelvinkuo** [comment]
k8s里强调的是万物皆资源，是不是可以把本文中的所有“对象”都替换成“资源”

**Comment by draveness** [comment]
> k8s里强调的是万物皆资源，是不是可以把本文中的所有“对象”都替换成“资源”

『万物皆资源』这句话是从哪里听来的...

**Comment by csh0101** [comment]
> @draveness
> > k8s里强调的是万物皆资源，是不是可以把本文中的所有“对象”都替换成“资源”
> 
> 『万物皆资源』这句话是从哪里听来的...

**Comment by csh0101** [comment]
> @draveness
> > 1.“Spec - 对象接口”：第一句话，多了一个“就是”
> > 2.“Spec - Pod”：“这些容器会共享网络和进程”，应该改成“这些容器会共享网络和存储”
> 
> 修复了，多谢

好像是在 k8s权威指南上看来的
